src/main.ts

```diff
@@ -63,7 +63,17 @@ async function start () {
   // start Vue application
   console.info('Starting app...') // eslint-disable-line no-console
   new Vue({
-    vuetify: new Vuetify({ iconfont: 'mdi' }),
+    vuetify: new Vuetify({
+      iconfont: 'mdi',
+      theme: {
+        themes: {
+          light: {
+            primary: '#1669bb', // same as $app-blue
+            error: '#d3272c'// same as $app-red
```

Comment:
Looks good. I'm happy to see more consistency in specifying and using colours.

Maybe add a comment against $app-blue and $app-red saying they're the same as the Vuetify theme primary/error colours? (So hopefully we will remember to change both files if needed.)

Reply:
Definitely! 

Will do, comment added for primary. The error colour i believe is a little different than the Vuetify theme, but it was the one selected by Tracy (The one we used in Name Request).